test/testutils.js

```diff
@@ -1,5 +1,7 @@
 "use strict";
 
+// disable eyeglass deprecations until tests are rewritten without deprecated methods
+process.env.EYEGLASS_DEPRECATIONS = "false";
```

Comment:
Are you ok with doing the option in a follow up PR? And I'll remove the environment variable in favor of it.